test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Get_ApplicationJson_Dictionary_Tests.cs

```diff
@@ -0,0 +1,107 @@
+        [DataRow("/get-applicationjson-dictionary-key", "get", "200")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationResponse(string path, string operationType, string responseCode)
+        {
+            var responses = this._doc["paths"][path][operationType]["responses"];
+
+            responses[responseCode].Should().NotBeNull();
+        }
+
+        [DataTestMethod]
+        [DataRow("/get-applicationjson-dictionary", "get", "200", "application/json")]
+        [DataRow("/get-applicationjson-dictionary-i", "get", "200", "application/json")]
+        [DataRow("/get-applicationjson-dictionary-ireadonly", "get", "200", "application/json")]
+        [DataRow("/get-applicationjson-dictionary-key", "get", "200", "application/json")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationResponseContentType(string path, string operationType, string responseCode, string contentType)
+        {
+            var content = this._doc["paths"][path][operationType]["responses"][responseCode]["content"];
+
+            content[contentType].Should().NotBeNull();
+        }
+
+        [DataTestMethod]
+        [DataRow("/get-applicationjson-dictionary", "get", "200", "application/json", "object")]
+        [DataRow("/get-applicationjson-dictionary-i", "get", "200", "application/json","object")]
+        [DataRow("/get-applicationjson-dictionary-ireadonly", "get", "200", "application/json",  "object")]
+        [DataRow("/get-applicationjson-dictionary-key", "get", "200", "application/json", "object")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationResponseContentTypeSchemaType(string path, string operationType, string responseCode, string contentType, string itemType)
+        {
+            var content = this._doc["paths"][path][operationType]["responses"][responseCode]["content"];
+
+            var schema = content[contentType]["schema"];
+            var type = schema["type"];
+            type.Value<string>().Should().Be(itemType);
+        }
+
+        [DataTestMethod]
+        [DataRow("/get-applicationjson-dictionary", "get", "200", "application/json", "object", "string")]
+        [DataRow("/get-applicationjson-dictionary-i", "get", "200", "application/json", "object", "integer")]
+        [DataRow("/get-applicationjson-dictionary-ireadonly", "get", "200", "application/json", "object", "number")]
+        [DataRow("/get-applicationjson-dictionary-key", "get", "200", "application/json", "object", "boolean")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationResponseContentTypeSchemaPropertiesType(string path, string operationType, string responseCode, string contentType, string dataType, string itemType)
+        {
+            var content = this._doc["paths"][path][operationType]["responses"][responseCode]["content"];
+
+            var schema = content[contentType]["schema"];
+            var additionaltype =schema["additionalProperties"]["type"];
```

Comment:
For readability, it's better to add a space before and after the equal sign.

Reply:
Updated!